core/src/main/res/drawable/drawer_list_selector_dark.xml

```diff
@@ -0,0 +1,4 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_activated="true" android:drawable="@drawable/activated_color_dark" />
```

Comment:
it would seam you can do this if you have the selector in res/color as shown in the example https://developer.android.com/guide/topics/resources/color-list-resource but implementing this in res/drawable this way leads to run time errors. Another option would be to make a res/color folder but that might be worse by confusing people since it seams we are storing all of the color values in res/value/color